CMakeLists.txt

```diff
@@ -31,10 +33,8 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") AND NOT (CMAKE_CXX_COMPILER_VER
 endif ()
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-aux_source_directory(./src SRC_LIST)
 file(GLOB_RECURSE SRC_LIST "./plugins/*.cpp" "./src/*.cpp")
```

Comment:
`SRC_LIST` is used in all other targets of cpp-ethereum repo. Also `eth_add_executable` expects `SRC_LIST` to be set. I would prefer to leave the current name for now.